adobe_vipm/adobe/client.py

```diff
@@ -355,22 +355,22 @@ def create_preview_order(
             "lineItems": [],
         }
 
-        for item in items:
-            product: AdobeProduct = self._config.get_adobe_product(item["productItemId"])
-            quantity = item["quantity"]
-            old_quantity = item["oldQuantity"]
+        for line in lines:
+            product: AdobeProduct = self._config.get_adobe_product(line["item"]["id"])
+            quantity = line["quantity"]
+            old_quantity = line["oldQuantity"]
 
             if quantity > old_quantity:
-                # For purchasing new items (oldQuantity = 0) or upsizing items
+                # For purchasing new lines (oldQuantity = 0) or upsizing lines
                 # quantity it must send the delta (quantity - oldQuantity) since
                 # it is placing a new order.
-                # For downsizing items quantity it must send the actual quantity
+                # For downsizing lines quantity it must send the actual quantity
                 # since the previous purchased quantity has been returned back
                 # through one or more RETURN orders.
                 quantity = quantity - old_quantity
             payload["lineItems"].append(
                 {
-                    "extLineItemNumber": item["lineNumber"],
+                    "extLineItemNumber": line["id"],
```

Comment:
According to [API docs](https://softwareone.atlassian.net/wiki/spaces/GPTEAM/pages/5222072917/Order+Object) line["id"] has the format ALI-1234-1234-1234-0001 but Adobe expect an integer, we should take just the sequential part of the ID here, right ?

Reply:
Yup, you are correct here, my friend. Good catch! :-)

Reply:
added `to_adobe_line_id`